symbols/ram.yml

```diff
@@ -472,6 +472,14 @@ ram:
         Bitset of enabled VRAM banks
         
         type: vram_banks_set
+    - name: THREAD_INFO_STRUCT
+      address:
+        EU: 0x22B9F88
+        NA: 0x22B9648
+      length:
+        EU: 0x4
+        NA: 0x4
```

Comment:
Shouldn't this be the size of `struct thread_info`? But also, since that's uncertain, it might be better to just omit the length here.

Reply:
Oh, right. I incorrectly wrote this down as a pointer at first but later corrected it. I must have forgotten to change the size.

> since that's uncertain, it might be better to just omit the length here.

Alright